### ISSUE TITLE
Creating a WEATHER compset to handle weather vs climiate namelist settings

### ISSUE THREAD

**Issue by MiCurry** [ESCOMP/CAM#441]
Continuing discussion from PR #433 in `bld/namelist_files/namelist_defaults_cam.xml` and 9/28/21 AME SE meeting. There are namelist default settings which, depending on the application weather vs climate, will be set conversely. For example, `scale_dry_air_mass` will be set to scale during climate simulations, but will not be set for weather predictions, which start from a reanalysis, like GFS output.

A few names for this compset:

* `FWEATHER` - But FW is already in use?
*  `Weather` - Straight and too the point..
* `Finitalize` - I'm not a big fan of this, as it doesn't seem to be associated with a 'weather' compset. Users might think its again running climate initialization
* `WFweather` - WF = Weather Forecasting. This could later be extended in `WFnudging` or `WFda` for weather runs with data analysis.
* Others?

Which components would the 'base' weather compset need to be coupled too? If any?

**Comment by cacraigucar** [comment]
My 2 cents worth...

CAM compsets are all prefixed with F (except for Q for aquaplanet and P for port).  

`WF` might be a good prefix, but would this be for Atmosphere only runs or coupled? If coupled weather runs will ever be made, I'd suggest leaving this prefix for those type of runs (to be equivalent to `B` compsets) and having an Fsomething as the CAM only.

What about a prefix of `FIW` where it could stand for "initialize weather"?  We could then have `FIW2000climo`, `FIWHIST` (if that makes sense), `FWIW2000climo` for WACCM runs, etc.

We then probably want to make a CAM compset longname qualifier `%IWTHR` or something like that, so that the compsets have something to actually key off of for namelist settings.  Current qualifiers can be seen by looking at the top of cime_config/config_component.xml.

**Comment by MiCurry** [comment]
@cacraigucar I like `FIW`. I think this inital compset would be non-coupled runs, but we would eventually use it as a base to create coupled compsets. I'm familar with this [compset list](https://www.cesm.ucar.edu/models/cesm2/config/compsets.html), but is there a list of what the first letter represents? Does the first letter (like `F`), just represent the dominate component?

But yes, I'm on board for `FIW`.

**Comment by cacraigucar** [comment]
> @cacraigucar I like `FIW`. I think this inital compset would be non-coupled runs, but we would eventually use it as a base to create coupled compsets. I'm familar with this [compset list](https://www.cesm.ucar.edu/models/cesm2/config/compsets.html), but is there a list of what the first letter represents? Does the first letter (like `F`), just represent the dominate component?
> 
> But yes, I'm on board for `FIW`.

Yes, the first letter represents the compset class (which component it belongs to).  `F` followed by anything else is recognized as a CAM compset.

**Comment by swrneale** [comment]
I think I would still worry about the WACCM confusion. What if WACCM wanted to do ICs in the same way; FWIW? Other components use letters pretty liberally in their compsets. How about FCAST or FMET or FIC?

**Comment by fischer-ncar** [comment]
Being a casual observer of this issue.  I will have to say that FMET stands out as the best to me.

**Comment by andrewgettelman** [comment]
I think FMET is probably a good option. This is for the short name of the compset.

What are we doing with the long name in the longer term?   Right now we have CAM60% and CAM%

I might propose that at some point this switch to SIMA% where we can have SIMA%WCTS (WACCM), SIMA%CAM63, SIMA%MET, SIMA%CAMDEV, SIMA%CAM7, etc.

Not sure when the best time for that to happen is. Maybe at the same time.

**Comment by nusbaume** [comment]
I would also vote for "FMET", as "FCAST" might get confused with a CAM-chem compset, and the rest might get confused with a WACCM compset.  I should note that there is an "FMOZ" compset, but it looks quite old and not frequently used, so I suspect there won't be any confusion there.  I also think we can use `%MET` as the qualifier in the long name as well.

@andrewgettelman In terms of SIMA, I might vote that we save that switch for CAMDEN, as that is the code base that is fully implementing all of the SIMA infrastructure (and will eventually replace CAM).  If that sounds reasonable to you then we can make an issue there and start implementing that change.
